src/Databricks/Az.Databricks.psd1

```diff
@@ -0,0 +1,133 @@
+#
+# Module manifest for module 'Az.Databricks'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 3/24/2020
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = './Az.Databricks.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
```

Comment:
Version usually starts from `0.1.0`
See https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase